spec/margin.md

```diff
@@ -50,15 +50,8 @@ Margin has a single account struct as state.
 pub struct MarginAccount {
```

Comment:
Note: this contract should have some state. For safety purposes we should not allow users to trade on any pair but only on pairs that meet a certain liquidity ratio.

Reply:
Are there metrics on volatility via the dex that we can easily query?

Reply:
we would need to figure out how to handle the calculations. In stocks, it is calculated based on open interest. We don't have the same on dexs. It would need to be calculated using information from multiple exchanges